12_glob_lab/globlab.txt

```diff
@@ -1,29 +1,59 @@
 Glob Lab.
 https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
-1. Type in a glob pattern to match:
+1. 
+
+Type in a glob pattern to match:
 
-  1. All files whose name starts with a lowercase letter
+ 
+ 1. All files whose name starts with a lowercase letter
   Answer: 
 
-  2. All files whose name starts with an uppercase letter
+ 
+ls -d [[:lower:]]*/*
+ 2. All files whose name starts with an uppercase letter
   Answer:
 
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
+ 
+ls -d [[:upper:]]*/* - for some reason shows the following:
+Doc/README.txt Doc/test.c
+Why test.c as well?
+
+ 3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
-  4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
+ 
+echo *[a,e,i,o,u]

```

Comment:
Better without the comma: `*[aeiou]`. A comma is a character like any other and so a file named `hi,` will match your pattern